tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1095,22 +1095,141 @@ public function test_build_line_items( bool $itemized = false, array $expected_i
+				'cart_product_type' => 'subscription',
+				'expected'          => false,
+			],
+			'pre-order in cart'                    => [
+				'feature_flag'      => true,
+				'is_checkout'       => true,
+				'adaptive_pricing'  => 'yes',
+				'cart_product_type' => 'pre-order',
+				'expected'          => false,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_is_adaptive_pricing_supported is not named in camelCase. (undefined)

(CamelCaseMethodName)